src/Microsoft.ML.AutoML/Utils/MLNetUtils/ArrayDataViewBuilder.cs

```diff
@@ -0,0 +1,468 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.AutoML
+{
+    using BitArray = System.Collections.BitArray;
+
+    /// <summary>
+    /// This is a class for composing an in memory IDataView.
+    /// </summary>
+    internal sealed class ArrayDataViewBuilder
```

Comment:
This appears to be duplicated from:

https://github.com/dotnet/machinelearning/blob/a802127883c0020516aa573436029cabed65d591/src/Microsoft.ML.Data/DataView/ArrayDataViewBuilder.cs#L14-L18

We shouldn't be doing that.

Reply:
yes, removed this one too.  I filed https://github.com/dotnet/machinelearning/issues/4064 the first time I looked at it, this time I have a bit more time so looking into what else can I remove without much code churn.